scripts/deblur

```diff
@@ -52,9 +57,19 @@ def deblur_cmds():
                    '(same for N indels)')
 @click.option('--indel-max', required=False, type=int, default=3,
               help="Maximal indel number")
+@click.option('--log-level', required=False, type=int, default=2,
```

Comment:
It it applies to other parameters (e.g. mean error and read error) please change.